ts/data.ts

```diff
@@ -1,8 +1,22 @@
 /* exported data */
+interface dataModel {
+  view: string;
+  entries: Entry[];
+  editing?: null;
+  nextEntryId: number;
```

Comment:
Why optional on your `editing` property? This property should always exist on your `dataModel`